.travis.yml

```diff
@@ -1,16 +1,48 @@
+dist: bionic
 language: python
 sudo: false
 cache:
+  pip: true
   directories:
   - eggs
-  - downloads
 python:
-  - 2.7
+  - "2.7"
 matrix:
+  include:
+    # - python: "2.7"
+    #   env: PLONE_VERSION=43
+    # - python: "2.7"
+    #   env: PLONE_VERSION=51
+    # - python: "2.7"
```

Comment:
I think the `-python "2.7"` line still needs to be there, otherwise the structure differs from the python3.7/plone52 below it.